chart/Chart.yaml

```diff
@@ -15,8 +15,8 @@
 
 apiVersion: v2
 name: dnation-kubernetes-monitoring
-version: 2.2.0
-appVersion: 2.2.0
+version: 2.2.3
```

Comment:
Maybe I would increase the `minor` instead of `patch` version. You are adding new dashboard, so in monitoring its like adding new functionality for me.